packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -5,10 +5,10 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v0.0.6
+    tag: v0.0.7
   securityScan:
     repository: rancher/security-scan
-    tag: v0.2.0
+    tag: v0.2.1
```

Comment:
are these images already pushed/mirrored in dockerhub ?

Reply:
yes they have been tagged and built- here https://github.com/rancher/security-scan/releases/tag/v0.2.1